docs/documentation/adding-css-javascript-and-images.md

```diff
@@ -37,23 +44,29 @@ Import this file into your `application.scss` file without the underscore:
 @import "admin";
 ```
 
-## JavaScript
+## Add JavaScript
 
-You can use JavaScript to make changes to a web page without loading a new one. For example a user could enter some numbers, then JavaScript displays the results of a calculation without loading a new page.
+You can use JavaScript to make changes to a web page without loading a new one. For example, a user could enter some numbers, then JavaScript displays the results of a calculation without loading a new page.
 
-To add JavaScript use:
+To add JavaScript, use:
 
 ```
 /app/assets/javascripts/application.js
 ```
 
 Do not edit the file `/public/javascript/application.js` because it’s deleted and rebuilt every time you make a change to your prototype.
 
-## Images
+To add a whole JavaScript file, add it to `app/assets/javascript`.
```

Comment:
Is it awkward to use 'add' twice in this sentence? Also, what does the user add the whole JS file to - the Kit?

Maybe we could write something like:

```suggestion
To add a whole JavaScript file to the Kit, run `app/assets/javascript/[your filename]`.
```